corehq/motech/repeaters/tests/test_proxy_models.py

```diff
@@ -346,21 +349,21 @@ class TestAppStructureRepeater(TestRepeaterModelsAttrEquality):
     def test_have_same_attrs(self):
         couch_attrs = self.get_cleaned_couch_attrs(AppStructureRepeater)
         sql_attrs = self.get_sql_attrs(SQLAppStructureRepeater)
-        self.assertEqual(couch_attrs ^ sql_attrs, set())
+        self.assertEqual(couch_attrs ^ sql_attrs, {'_all_option_value_fields'})
```

Comment:
It seems that `'_all_option_value_fields'` should not be returned by `get_sql_attrs(...)`. Would it make sense to have `get_sql_attrs` filter that one out, and then change these tests back to the way they were before?